esp/src/src/DiskUsage.ts

```diff
@@ -1,14 +1,14 @@
 import { Gauge } from "@hpcc-js/chart";
 import { Palette } from "@hpcc-js/common";
-import { GetTargetClusterUsageEx, MachineService, DFUXRefService, WsDFUXRef } from "@hpcc-js/comms";
+import { WsMachineEx, MachineServiceEx, DFUXRefService, WsDFUXRef } from "@hpcc-js/comms";
 import { ColumnFormat, Table } from "@hpcc-js/dgrid";
 import { FlexGrid } from "@hpcc-js/layout";
 import * as dojoOn from "dojo/on";
 import nlsHPCC from "./nlsHPCC";
 
 Palette.rainbow("DiskUsage", ["green", "green", "green", "green", "green", "green", "green", "green", "orange", "red", "red"]);
 
-const machineService = new MachineService({ baseUrl: "", timeoutSecs: 360 });
+const machineService = new MachineServiceEx({ baseUrl: "", timeoutSecs: 360 });
```

Comment:
Looking at this code - I think we need to preserve MachineService as the extended class name (which implies the wsdl generated code should generate `MachineServiceBase` or similar for the generated class?

Reply:
The most recent deployment for hpcc-js cleared up the GH action errors & would include this change.